src/gui/accountstate.h

```diff
@@ -184,6 +191,8 @@ public slots:
     void setState(State state);
     void fetchNavigationApps();
 
+    void setRetryCount(int count);
```

Comment:
@mgallien Why do you need to have this method with parameters? Isn't just having `resetRetryCount` going to be enough? I don't see you using this setter with values other than `0`. Also, this slot is public, even though it is only used internally. Maybe you can just assign directly `_retryCount = 0;`?

Reply:
changed it to `resetRetryCount`
it was already private so no change in visibility